vulkaninfo/vulkaninfo.c

```diff
@@ -4558,11 +4558,6 @@ static void AppGpuDumpQueueProps(const struct AppGpu *gpu, uint32_t id, FILE *ou
         printf("\t\t\t\"queueCount\": %u,\n", props.queueCount);
         printf("\t\t\t\"queueFlags\": %u,\n", props.queueFlags);
         printf("\t\t\t\"timestampValidBits\": %u,\n", props.timestampValidBits);
```

Comment:
The comma after `timestampValidBits\": %u` makes this invalid JSON.

Reply:
Fixed. **all** validators seem to pass now.